pkg/kubectl/cmd/update.go

```diff
@@ -60,6 +63,10 @@ func NewCmdUpdate(f *cmdutil.Factory, out io.Writer) *cobra.Command {
 	cmd.MarkFlagRequired("filename")
 	cmd.Flags().String("patch", "", "A JSON document to override the existing resource. The resource is downloaded, patched with the JSON, then updated.")
 	cmd.MarkFlagRequired("patch")
+	cmd.Flags().Bool("force", false, "Delete and re-create the specified resource")
+	cmd.Flags().Bool("cascade", true, "Only relevant during a force update. If true, cascade the delete of resources managed by this resource (e.g. Pods created by a ReplicationController).  Default true.")
```

Comment:
I feel like this flag should prompt a warning. We tell users to use `rolling-update` for destructive updates on replicated pods. If we also allow them to update rcs with `kubectl update --force` we probably want to make sure they actually did want to delete all replicas at once and recreate them.